brave/brave-ios
# Fix #6361: New NFT detail

## Summary of Changes
New design for NFT (currently only for ERC 721 token)


This pull request fixes #6361

## Submitter Checklist:


## Test Plan:
1. Check ERC 721 token detail is using the new design
2. Check ERC 721 token details renders correct nft information 
3. Check ERC 721 token id can redirect users to the explorer site
4. Check any other NFTs details will use the same token details minus price component
5. Check rest token details stay the same


## Screenshots:


NFT has image | NFT has no image
--|--
![Simulator Screen Shot - iPhone 14 Pro - 2022-11-23 at 17 42 35](https://user-images.githubusercontent.com/1187676/203658751-0b2f2525-eb9b-4843-969b-c3192d001d73.png) | ![Simulator Screen Shot - iPhone 14 Pro - 2022-11-23 at 17 42 22](https://user-images.githubusercontent.com/1187676/203658780-20f37bff-1b87-4465-8bbc-cb92ad39dab4.png)

## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
